### DISCUSSION TITLE
Does next-api-decorators work with the newer API Routes in Next.js 13 (Beta)?

### DISCUSSION THREAD

**Discussion by icazemier** [instantcommerce/next-api-decorators#568] [Q&A]
Please refer to: https://beta.nextjs.org/docs/upgrade-guide#api-routes

Thanks for the great library!

**Comment by sebpowell** [comment]
@icazemier – tried it this evening, and it doesn't look like it works – unless I'm not using it properly.

**Reply by icazemier** [reply]
That's a proper answer to my question, but the context of the question is, that I would dearly have api-decorators compatible with the new api-routes 😉.

No it does not work as you mentioned, but "will it in the future" would be the right question.

My bad.
